src/util.js

```diff
@@ -1,17 +1,28 @@
-// A bunch of helper functions.
+import {
```

Comment:
@vojtajina For some reason my import is consistently importing undefined. Am I doing something wrong?